README.adoc

```diff
@@ -1,17 +1,22 @@
+:projectKey: optaweb-employee-rostering
+:sonarBadge: image:https://sonarcloud.io/api/project_badges/measure?project={projectKey}
+:sonarLink: link="https://sonarcloud.io/dashboard?id={projectKey}"
+
 = Employee Rostering Optimization as a Service
 
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=alert_status[
-"Quality Gate Status", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=reliability_rating[
-"Reliability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=security_rating[
-"Security Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=sqale_rating[
-"Maintainability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=ncloc[
-"Lines of Code", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=coverage[
-"Coverage", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://travis-ci.com/kiegroup/optaweb-employee-rostering.svg?branch=master[
+"Build Status", link="https://travis-ci.com/kiegroup/optaweb-vehicle-routing"]
+image:https://img.shields.io/badge/stackoverflow-ask_question-orange.svg?logo=stackoverflow[
+"Ask question on Stack Overflow", link="https://stackoverflow.com/questions/tagged/optaplanner"]
```

Comment:
Wrong link (optaweb-employee-rostering has it own tag: https://stackoverflow.com/questions/tagged/optaweb-employee-rostering)